### PR TITLE
DEV: Skip flaky system tests on CI

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: spec/system/user_preferences_interface_spec.rb [hunk 1]
```diff
@@ -9,6 +9,15 @@
 
   describe "Bookmarks" do
     it "changes the bookmark after notification preference" do
+      skip(<<~SQL) if ENV["CI"]
```
Comment (nbianca) [inline spec/system/user_preferences_interface_spec.rb:12]: The `heredoc` label is a bit odd. The text is not anything related to SQL.

Comment (nbianca) [reply spec/system/user_preferences_interface_spec.rb:12]: Also, long term this should be fixed. I am wondering why this is failing.

**Review by nbianca — APPROVED** [review]
Left just a small comment. :+1:

File: spec/system/user_preferences_interface_spec.rb (hunk 1 above)
Comment (tgxworld) [reply spec/system/user_preferences_interface_spec.rb]: Good catch! I've fixed the heredoc.

**Comment by tgxworld** [comment]
Thank you for reviewing @nbianca

**Pull request merged**